portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,40 @@
+<html>
+
+<head>
+	<title>LightScape Gallery</title>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div class="topnav">
+		<a href="index.html">Home</a>
+		<a class="active" href="gallery.html">LightScape Gallery</a>
+	</div>
+	<div class="fadein" id="lightscape">
+		<h1>Chicago's LightScape</h1>
+		<p>The Lightscape at Chicago is a yearly event that is hosted during the
+			months of December and January. It takes place in the Chicago Botanic
+			Gardens. The whole Lightscape takes about an hour to go through, but
+			considering how the trail is only one mile long, you can imagine that
+			most people take their time to take in all the beautiful views. You can
+			find the official website
+			<a href="https://chicagobotanic.org/lightscape" style="color:white">here</a>
+			.</p>
+	</div>
+	<div class="gallery">
+		<a href="images/lightscape1.jpg"><img src="images/lightscape1.jpg"></a>
```

Comment:
Formatting looks very weird here.  Did an autoformatter do this or is there a reason there are so many tabs?

Also, I think you should covert all tabs to spaces in these files

Reply:
Hmm I didn't notice. I'm pretty sure the formatter in the editor did this, so I will get to work on fixing that. Thanks!